### ISSUE TITLE
When splitting an experiment, resulting names of splits can exceed 255 characters

### ISSUE THREAD

**Issue by arteymix** [PavlidisLab/Gemma#876]
For GSE224096:

```
2023-09-28 09:48:16,803 WARN 53995 [main] org.hibernate.engine.jdbc.spi.SqlExceptionHelper.logExceptions(144) | SQL Error: 1406, SQLState: 22001 2023-09-28 09:48:16,804 ERROR 53995 [main] org.hibernate.engine.jdbc.spi.SqlExceptionHelper.logExceptions(146) | Data truncation: Data too long for column 'NAME' at row 1 2023-09-28 09:48:16,906 ERROR 53995 [main] ubic.gemma.core.util.AbstractCLI.executeCommand(136) | splitExperiment failed. org.springframework.dao.DataIntegrityViolationException: could not execute statement; SQL [n/a]; nested exception is org.hibernate.exception.DataException: could not execute statement 
```

Reference: https://github.com/PavlidisLab/GemmaCuration/issues/421

**Comment by arteymix** [comment]
We should truncate before abbrevieating the original name. Tis experiment has ellipsis after a space: https://gemma.msl.ubc.ca/expressionExperiment/showExpressionExperiment.html?id=30985

**Comment by arteymix** [comment]
Fully fixed by 4242e83ceb6ea01d50e1b743486eb8965d7d5dc7
